base/applications/osk/lang/it-IT.rc

```diff
@@ -129,13 +129,13 @@ END
 
 IDD_WARNINGDIALOG_OSK DIALOGEX 0, 0, 250, 97
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_POPUP | WS_CAPTION
-CAPTION "On-Screen Keyboard"
+CAPTION "Tastiera visivo"
```

Comment:
Better "Tastera a schermo". Is a more letteral and correct translation in my opinion,

Reply:
The title of the application is the same even on main dialog box and for this reason I have kept the same tile for the sake of convenience to avoid differences between words.

I could change it to `tastiera a schermo` nonetheless, I need to replace the main dialog's title at the top as well.

Reply:
Yes. I think is better to consolidate changing the main dialog's title too.